sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.getAllInProgressApplications.e2e-spec.ts

```diff
@@ -0,0 +1,204 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  authorizeUserTokenForLocation,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+  getAuthRelatedEntities,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import {
+  createFakeInstitutionLocation,
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeApplicationOfferingRequestChange,
+} from "@sims/test-utils";
+import {
+  ApplicationOfferingChangeRequestStatus,
+  InstitutionLocation,
+} from "@sims/sims-db";
+import { FieldSortOrder, getISODateOnlyString } from "@sims/utilities";
+import { getUserFullName } from "../../../../utilities";
+
+describe("ApplicationOfferingChangeRequestAESTController(e2e)-getAllInProgressApplications", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeFLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
```

Comment:
Let me know if I am missing something. Do we need collegeF or getAuthRelatedEntities for any particular reason here ?

getAuthRelatedEntities  is to get user associated to the token type.

Reply:
I removed everything related to institutions from the test. Thanks.